library/blockly/html/blocks.js

```diff
@@ -649,6 +1070,209 @@ Blockly.Blocks['bordercol'] = {
 	}
 };
 
+//Border radius
+Blockly.Blocks['borderrad'] = {
+	init: function(){
+		this.jsonInit({
+			"message0": 'border-radius: %1 ;',
+			"args0": [
+				{
+					"type": "field_input",
+					"name": "content",
+					"text": "10px"
+				}
+			],
+			"previousStatement": "stylecontent",
+			"nextStatement": "stylecontent",
+			"colour": 290,
+			"tooltip": "CSS Border Radius",
+			"helpUrl": "https://www.w3schools.com/cssref/pr_border-radius.asp"
+		});
+	}
+};
+
+//Cursor
+Blockly.Blocks['cursor'] = {
+	init: function() {
+		this.jsonInit({
+			"message0": 'cursor: %1 ;',
+			"args0": [
+				{
+					"type": "field_dropdown",
+					"name": "content",
+					"options": [
```

Comment:
![image](https://user-images.githubusercontent.com/32415108/34533459-36ece4a2-f0b2-11e7-9d49-96e47371f820.png)

Please change to only include these options.